### DISCUSSION TITLE
Create Reverse Alias via Extension

### DISCUSSION THREAD

**Discussion by LenKiMo** [simple-login/app#930] [Feature Requests]
At the moment, the typical way I want to send a message by hosting the Alias address on Simplelogin is to open the SimpleLogin Web App, find the Alias I need to be the sender, and increase the Reverse Alias under this Alias, the Reverse Alias that you obtain is then filled in to the recipient of the message.

This process required me to switch pages and navigate through multiple interfaces on the SimpleLogin Web, which was, to be honest, inconvenient. So I wanted SimpleLogin to add the ability to create Reverse aliases in the browser extension so that users didn't have to switch to another window to create the required Reverse Alias

Or, this process could have been less complicated than what I am doing now. SimpleLogin supports using aliases to send outgoing emails in a more convenient way. If so, what would a more typical operational flow look like?

**Comment by nguyenkims** [comment]
@LenKiMo you can now create a reverse alias directly from the Chrome/Firefox extension.

**Reply by tangiblehologram** [reply]
Creating a reverse alias is a bit better from the extension but should be made easy.
